audbackend/core/backend.py

```diff
@@ -652,13 +663,6 @@ def versions(
             ['1.0.0', '2.0.0']
 
         """
-        utils.check_path_for_allowed_chars(path)
-
-        vs = utils.call_function_on_backend(
-            self._versions,
-            path,
-            suppress_backend_errors=suppress_backend_errors,
-            fallback_return_value=[],
-        )
-
-        return audeer.sort_versions(vs)
+        paths = self.ls(path, suppress_backend_errors=suppress_backend_errors)
+        vs = [v for _, v in paths]
+        return vs
```

Comment:
As we now use `self.ls()` you should delete the definition of `_versions()` done a few lines before this method.

Reply:
Thanks, I forgot to remove it. Done.